pom.xml

```diff
@@ -264,7 +264,7 @@
         <junit.version>4.11</junit.version>
         <metrics-clojure.version>2.5.1</metrics-clojure.version>
         <hdrhistogram.version>2.1.7</hdrhistogram.version>
-        <calcite.version>1.4.0-incubating</calcite.version>
+        <calcite.version>1.9.0</calcite.version>
         <janino.version>2.7.8</janino.version>
```

Comment:
We will follow Janino from the Calcite dependency. I'll check it again and delete if it's not used anymore.